Library/Homebrew/brew.sh

```diff
@@ -32,7 +32,7 @@ onoe() {
   fi
   if [[ $# -eq 0 ]]
   then
-    /bin/cat >&2
+    cat >&2
```

Comment:
You can submit this change to Homebrew/brew.

Reply:
https://github.com/Homebrew/brew/pull/3942